LibOS/shim/src/fs/shim_fs.c

```diff
@@ -185,40 +185,31 @@ static int __mount_one_other (const char * key, int keylen)
 
 static int __mount_others (void)
 {
-    int ret = 0;
-    char * keybuf = NULL;
-
     if (!root_config)
-        goto out;
+        return 0;
 
     int nkeys;
     ssize_t keybuf_size;
     keybuf_size = get_config_entries_size(root_config, "fs.mount");
     if (keybuf_size < 0)
-        goto out;
+        return 0;
 
-    keybuf = malloc(keybuf_size);
-    if (!keybuf) {
-        ret = -ENOMEM;
-        goto out;
-    }
+    char * keybuf = __alloca(keybuf_size);
```

Comment:
Please revert this change. Potentially large, user-controlled-size buffers should never be allocated with `alloca`.